packages/fluent-editor/src/modules/custom-clipboard.ts

```
@@ -23,6 +23,7 @@ export class CustomClipboard extends Clipboard {
   declare quill: FluentEditor
 
   prepareMatching(container: HTMLElement, nodeMatches) {
+    console.log(1111, 'prepareMatching');
```

ESLint

[error] Unexpected console statement. Only these console methods are allowed: warn, error.

(no-console)